Description: Upstream patch to re-add mouse click support
Author: Michael Lamothe <michael.lamothe@gmail.com>
Applied-Upstream: http://bazaar.launchpad.net/~lamothe/gnome-media-player/0.2-new/revision/112
--- a/src/vlc_engine.cc
+++ b/src/vlc_engine.cc
@@ -57,7 +57,7 @@
 	{
 		g_debug("linear deinterlacer");
 		vlc_argv[i++] = "--vout-filter=deinterlace";
-		vlc_argv[i++] = "--deinterlace-mode=linear";
+		vlc_argv[i++] = "--deinterlace-mode=blend";
 	}
 	else
 	{
@@ -65,8 +65,9 @@
 	}
 
 	instance = libvlc_new (i, vlc_argv);
-
+	
 	media_player = libvlc_media_player_new(instance);
+	libvlc_video_set_mouse_input(media_player, false);
 	libvlc_audio_set_volume(media_player, DEFAULT_VOLUME);
 
 	event_manager = libvlc_media_player_event_manager(media_player);
